third_party/custom_crt0/startup_stm32f407xx_stripped.s

```diff
@@ -0,0 +1,149 @@
+/**
+  ******************************************************************************
+  * @file      startup_stm32f407xx.s
```

Comment:
```suggestion
 * @file      startup_stm32f407xx_stripped.s
```

A general remark, should we rename the file to `startup_stm32f4xx.s`? No real need to name it strict but also no need to name it 407. Maybe 4xx is the better choice?

Reply:
Sounds reasonable. I'm assuming the reference in the README to the `stm32f407` should be turned into `stm32f4xx` aswell then?

Reply:
If you rename so `stm32f4xx`, please also change it there accordingly.